packages/experimental/Switch/src/Switch.styling.ts

```diff
@@ -34,24 +34,14 @@ export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, Sw
           alignItems: 'center',
           flexDirection: tokens.flexDirection,
           alignSelf: 'flex-start',
-          minHeight: tokens.minHeight,
-          minWidth: tokens.minWidth,
+          width: tokens.width,
           borderColor: tokens.focusStrokeColor,
           borderWidth: tokens.focusBorderWidth,
           borderRadius: tokens.focusBorderRadius,
           ...layoutStyles.from(tokens, theme),
```

Comment:
Shouldn't these be part of `...layoutStyles` ?

Reply:
minWidth, and minHeight are. Width currently isn't, but it probably should be.

Reply:
Yeah you can add it. It's mostly a list of tokens that have manually been added over time, there isn't really a reason why a particular token has been left out.